sirepo_bluesky/tests/test_converter.py

```diff
@@ -0,0 +1,62 @@
+import json
+from pprint import pprint
+
+import pytest
+import yaml
+
+from sirepo_bluesky.utils.json_yaml_converter import dict_to_file, json_to_yaml, yaml_to_json
+
+
+def test_dict_to_file(tmp_path):
+    test_dict = {"a": 1, "b": 2, "c": 3}
+
+    dict_to_file(tmp_path / "test1.json", test_dict)
```

Comment:
Looking at the usage in the test, it prompts for a swap of the input positional arguments. The function is `dict_to_file` hinting the first input should be a dictionary, and the second one be the target file. Can we swap the parameters please to avoid confusion with the usage?